converter/src/test/java/gov/cms/qpp/conversion/validate/CpcClinicalDocumentValidatorTest.java

```diff
@@ -140,17 +141,24 @@ void testCpcPlusSubmissionBeforeEndDate() {
 		System.clearProperty(CpcClinicalDocumentValidator.END_DATE_VARIABLE);
 	}
 
-	@Test
-	void testCpcPlusSubmissionAfterEndDate() {
+	@ParameterizedTest
+	@CsvSource({"meep@mawp.blah, meep@mawp.blah", ", cpcplus@telligen.com"})
+	void testCpcPlusSubmissionAfterEndDate(String systemValue, String expected) {
 		LocalDate endDate = LocalDate.now().minusYears(3);
+		String formattedDate = endDate.format(CpcClinicalDocumentValidator.END_DATE_FORMAT);
+		if (systemValue != null) {
+			System.setProperty(CpcClinicalDocumentValidator.CPC_PLUS_CONTACT_EMAIL, systemValue);
+		}
 		System.setProperty(CpcClinicalDocumentValidator.END_DATE_VARIABLE, endDate.toString());
 		Node clinicalDocument = createValidCpcPlusClinicalDocument();
+
 		cpcValidator.internalValidateSingleNode(clinicalDocument);
 
 		assertThat(cpcValidator.getDetails())
 			.comparingElementsUsing(DetailsErrorEquals.INSTANCE)
-			.containsExactly(ErrorCode.CPC_PLUS_SUBMISSION_ENDED.format(endDate));
+			.containsExactly(ErrorCode.CPC_PLUS_SUBMISSION_ENDED.format(formattedDate, expected));
 		System.clearProperty(CpcClinicalDocumentValidator.END_DATE_VARIABLE);
+		System.clearProperty(CpcClinicalDocumentValidator.CPC_PLUS_CONTACT_EMAIL);
```

Comment:
this isn't guaranteed to run -- maybe put this in an @AfterEach

Reply:
+1 on this.

Reply:
Can also use maven surefire plugin it include a system property.

Reply:
I agree as well.  It would be good to consolidate any instances of clearing the system properties to an @aftereach across all the tests here.